proposals/NNNN-atf-script-runner.md

```diff
@@ -0,0 +1,43 @@
+# ATF Script Runner
+
+* Proposal: [SDL-NNNN](NNNN-atf-script-runner.md)
+* Author: [Dmytro Boltovskyi](https://github.com/dboltovskyi)
+* Status: **Awaiting review**
+* Impacted Platforms: [ATF]
+
+## Introduction
+
+Originally ATF was developed for SDL developers and thats why it lacks a few features of modern Test Frameworks
```

Comment:
@dboltovskyi  for SDL developers? Not for testers? :)

Reply:
@dboltovskyi ,
no matter for who ATF is, but I suggest to do not mention other modern framwork.
It could birth bad thoughts about usage of some other tool instead of ATF.
Please rephrase it in manner like  - ATF is a great tool fpor booth developers and testers that help test any change in SDL.

Reply:
@AKalinich-Luxoft , @AByzhynar Actually I'm not aware about this history of ATF, but this particular phrase was provided by Alex Kutsan :)